src/e2e/pages/landingPage.ts

```diff
@@ -262,9 +262,10 @@ export class LandingPage {
   }
 
   async goToSignIn() {
+    await this.page.waitForTimeout(500); // sign in button may not be loaded at this point.
```

Comment:
I think setting the timeout directly as an option on the `click` method would also work:

```suggestion
    await this.signInButton.click({ timeout: 500 });
```

Reply:
k let me try this locally first